tools/nessus/nessus.md

```diff
@@ -137,9 +146,40 @@ Set-SmbServerConfiguration -AutoShareServer $False -AutoShareWorkstation $False
 
 ### Scanning Linux
 
-**Linux** does support credentialed scans via SSH key authentication.
+**Linux** does support credentialed scans via SSH key authentication.[^linux-scan] The account used for scanning must have root privileges.
+
+*Notice: If the shell variable PS1 is shorter than 4 letters, the scan time may drastically increase.*[^ps1]
+
+#### Generate SSH public and private key pair
+
+It does not matter where and how the key pair is created. However, it is important that the defined Tenable Nessus user owns the keys.
+
+Create a key pair with `ssh-keygen -t ed25519`. It is recommended to set **nessus** as the filename. Then, choose a passphrase (or choose an empty password by pressing `Return` twice). If you specify a passphrase, you must specify it in **Policies** > **Credentials** > **SSH settings** for Tenable Nessus to use key-based authentication.
+
+Upon creating a user account which Nessus uses to login on the remote system, make sure that the account has no valid password set. On Linux systems, new user accounts are locked by default, unless you explicitly set an initial password. If you are using an account where someone had set a password, use the `passwd –l <username>` command to lock the account.
```

Comment:
Add information on creating a user with `sudo` permission:

- [How To Create A New Sudo Enabled User on Ubuntu](https://www.digitalocean.com/community/tutorials/how-to-create-a-new-sudo-enabled-user-on-ubuntu)
- [How To Edit the Sudoers File](https://www.digitalocean.com/community/tutorials/how-to-edit-the-sudoers-file)
- [SSH Privilege Escalation](https://community.tenable.com/s/article/SSH-Privilege-Escalation?language=en_US)